icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -112,7 +112,6 @@ class IntegerWidth;
 
 namespace impl {
 
-#ifndef U_HIDE_INTERNAL_API
```

Comment:
Thanks for fixing this. Though I think this was fixed in the maint-64 branch during the 64.2 update. You might need to rebase and resolve conflicts.

Reply:
yes, I just noticed, rebased.